docs/copied-from-beats/docs/shared-path-config.asciidoc

```diff
@@ -106,3 +106,20 @@ Example:
 ------------------------------------------------------------------------------
 path.logs: /var/log/beats
 ------------------------------------------------------------------------------
+
+[float]
+==== `system.hostfs`
+
+Specifies the mount point of the host's filesystem for use in monitoring a host.
+This can either be set in the config, or with the `--system.hostfs` CLI flag. This is used for cgroup self-monitoring.
+ifeval::["{beatname_lc}"=="metricbeat"]
+This is also used by the system module to read files from `/proc` and `/sys`.
+endif::[]
+
+
+Example:
+
+[source,yaml]
+------------------------------------------------------------------------------
+path.logs: /var/log/beats
+------------------------------------------------------------------------------
```

Comment:
this example doesn't match the section

Reply:
Ah, good spot 🎯. I'll follow up!

Reply:
Fixed in Beats. Copy/pasting into this PR so we don't have to wait.